app/models/merchant.rb

```diff
@@ -1,7 +1,8 @@
 class Merchant < ApplicationRecord
   enum :status, {"enabled" => 0, "disabled" => 1}
 
-  validates_presence_of :name
+  validates_presence_of :name, presence: true
+  # validates_presence_of :status, presence: true
```

Comment:
We will want this validation I believe since we added a default status for all merchants into our schema now